mods/gen4/formats-data.js

```diff
@@ -660,7 +660,7 @@ exports.BattleFormatsData = {
 		tier: "UU",
 	},
 	glaceon: {
-		randomBattleMoves: ["icebeam", "hiddenpowerground", "shadowball", "batonpass", "wish"],
+		randomBattleMoves: ["icebeam", "hiddenpowerground", "hiddenpowerfire", "shadowball", "batonpass", "wish", "substitute", "toxic", "protect"],
```

Comment:
I see potential for bad sets here. I mean it depends on if you think mono-attacking Glaceon is bad.